examples/step-74/doc/results.dox

```diff
@@ -0,0 +1,195 @@
+<h1>Results</h1>
+
+The output of this program consist of the console output and
+solutions in vtu format.
+
+Convergence rate for the smooth case with polynomial degree 3:
+@code
+DEAL::Cycle 0
+DEAL::Number of active cells:       16
+DEAL::Number of degrees of freedom: 256
+   Solving system...
+DEAL::Writing solution to <sol_Q3-00.vtu>
+   Error in the L2 norm       :     0.00193285
+   Error in the H1 seminorm   :     0.106087
+   Error in the enery norm    :     0.150625
+
+DEAL::Cycle 1
+DEAL::Number of active cells:       64
+DEAL::Number of degrees of freedom: 1024
+   Solving system...
+DEAL::Writing solution to <sol_Q3-01.vtu>
+   Error in the L2 norm       :     9.60497e-05
```

Comment:
This back and forth with the indentation is annoying and makes it hard to read. It's one of the reasons why we don't use `deallog` much any more.